### ISSUE TITLE
world finding problem

### ISSUE THREAD

**Issue by IlGab23** [Imprex-Development/orebfuscator#265]
### Description

i'm using a magma server type that can make forge and bukkit/spigot works together. The problem is that the forge don't create "world_nether" and "world_the_end" folders but it makes DIM-1(nether) and DIM1(end) in the world folder so the plugin can't find them and it works only in overworld. Is there any way to make the plugin finds the nether and the end?

### Reproduction Steps

none

### Expected Behaviour

none

### Server Details

Magma version 1.18.2-6f06aa3a (Implementing API version 1.18.2-R0.1-SNAPSHOT, Forge Version 40.2.0)

### Orebfuscator Version

5.3.2

### ProtocolLib Version

4.8.0

**Comment by NgLoader** [comment]
I'll take a look at this problem tomorrow

Have you already tried to change the world names in the config file from ``world_nether`` to ``nether``?

**Comment by IlGab23** [comment]
Of course, I've tried lots of combinations like "nether" or "world\DIM-1" or just "world" but nothing.

**Comment by NgLoader** [comment]
What you can do is to merge everything from nether and end into the world config section.
Orebfuscator is unable to detect any other worlds because of magma is just telling us that one world is registered with the bukkit api.

**Comment by Ingrim4** [comment]
Problem boils down to Magma not registering other worlds in their Bukkit-API implementation. This means Orebfuscator as well as any other plugin won't be able to differentiate the `overworld` from any other dimension. As Nils said you could merge all the world config sections into the default `overworld` section but you will have to disable the cache since Orebfuscator won't be able to tell what chunk belongs to what dimension which will end up poisoning the cache and ruin the performance (cache misses).

TL;DR: Magma needs to fix their Bukkit-API implementation nothing we can do about it.

**Comment by IlGab23** [comment]
ok. thanks for your support guys
